tests/test_pfs.py

```diff
@@ -39,15 +39,15 @@ def test_delete_non_existent_repo():
     assert len(client.list_repo()) == orig_repo_count
 
 
-def test_delete_all_repos():
-    client = python_pachyderm.Client()
+#def test_delete_all_repos():
```

Comment:
We should just remove these if they are not being used for testing.

Reply:
@actgardner Thanks for answering all of these questions—it looks like this function is still in the PFS mixin. Does the test not work in 2.0 or is it just flaky or something?

If the issue is just that it's flaky, we should add a todo and fix it. Otherwise it seems like we should remove the corresponding method from the client